src/module_EARTH_GRID_COMP.F90

```diff
@@ -256,10 +256,12 @@ SUBROUTINE EARTH_REGISTER(EARTH_GRID_COMP,RC_REG)
         specLabel=Driver_label_SetRunClock, specRoutine=NUOPC_NoOp, rc=RC_REG)
       ESMF_ERR_RETURN(RC,RC_REG)
       
+#if 0
```

Comment:
If this call is not needed anymore, delete it.

Reply:
We will keep same version as current GFSv16 production branch and remove it in a later commit (possibly s2s lib updates).